### PR TITLE
fix: Updated theme to latest version of Chakra and Frontity

### PR DESCRIPTION
**Pull request by Bowriverstudio**
Hello,

Chakra and Frontity both have changed a fair bit since this theme was created.  This Pull Request - updates all the packages to their latest (Feb 3rd, 2021) and follows the upgrade guidelines.

This is my first time submitting a pull request so if you would like additional changes please let me know.

Cheers,
Maurice

### CODE REVIEW

File: packages/frontity-chakra-theme/package.json [hunk 1]
```diff
@@ -20,14 +20,5 @@
...
     "access": "public"
-  },
-  "dependencies": {
-    "@chakra-ui/core": "0.5.0",
-    "@frontity/components": "^1.1.14",
-    "@frontity/hooks": "1.1.13",
-    "@frontity/html2react": "^1.1.14",
-    "emotion-theming": "10.0.19",
-    "frontity": "^1.4.2",
-    "react-icons": "3.8.0"
```
Comment (luisherranz) [inline packages/frontity-chakra-theme/package.json]: @Bowriverstudio, why did you move all the dependencies of the theme to the root `package.json`?

File: packages/frontity-chakra-theme/src/components/featured-post/featured-post.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const SecondaryPostPreview = ({ data, ...props }) => {
         <PostContent padding="40px" textAlign="left" mt="0">
           <PostCategories justifyContent="flex-start" categories={categories} />
           <PostTitle as="h2" mt="auto" pt="40px" fontSize="1.65rem">
-            {title}
+            {title} ss
```
Comment (luisherranz) [inline packages/frontity-chakra-theme/src/components/featured-post/featured-post.js]:
```suggestion
            {title}
```

File: packages/frontity-chakra-theme/src/components/loading.js [hunk 3]
```diff
@@ -1,6 +1,5 @@
-/**@jsx jsx */
-import { Box, CircularProgress } from "@chakra-ui/core";
-import { jsx } from "@emotion/core";
+import React from "react";
```
Comment (luisherranz) [inline packages/frontity-chakra-theme/src/components/loading.js]:
```suggestion
```

These React imports are not required in the latest version of Frontity anymore.

File: packages/frontity-chakra-theme/src/components/post/post-progressbar.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
```
Comment (luisherranz) [inline packages/frontity-chakra-theme/src/components/post/post-progressbar.js]:
```suggestion
```

**Review by luisherranz — APPROVED** [review]
@Bowriverstudio looks great, amazing work! 😄👏👏

I only have the question about the dependencies and a couple of suggestions.

---

@Segunadebayo, I think this is great. Would you please take a look as well, and if you think so, release a new version?

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "my-frontity-project",
-  "version": "1.0.0",
+  "version": "1.2.0",
```
Comment (luisherranz) [inline package.json]:
```suggestion
  "version": "1.0.0",
```

Oh, @Bowriverstudio you can remove this change and add a changeset using `npx changeset` in your local project. 

The version will be updated manually once we do the release, don't worry about that.

You can read more about changesets [here](https://docs.frontity.org/contributing/code-contribution-guide#changelogs).

File: packages/frontity-chakra-theme/package.json (hunk 1 above)
Comment (Bowriverstudio) [reply packages/frontity-chakra-theme/package.json]: @luisherranz, I'll move them back as I suspect it is better technically.

A few months ago, I included the package `frontity-contact-form-7` and at the time it was using a different version of frontity than myself.  This caused a conflict, which was resolved when I updated the  `frontity-contact-form-7` package.  I also noticed that if I moved the dependencies into the root I did not have maintain them.

I've created several common packages for my frontity projects (just not told mentioned them yet) and some of the more stable ones are: https://github.com/Bowriverstudio/frontity-microsoft-clarity, https://github.com/Bowriverstudio/frontity-menu, https://github.com/Bowriverstudio/frontity-gutenberg-styles

I just found it tedious to update all the packages individually and easier to have them in the root.   I suspect there is probably a better solution perhaps with lerna but I did not explore that too much.

Comment (luisherranz) [reply packages/frontity-chakra-theme/package.json]: Ok, thanks a lot for the explanation. This is something that we have to look into. Maybe we need a `npx frontity update` tool that makes it automatic.

**Comment by luisherranz** [comment]
@segunadebayo everything good here 👍 🙂

**Comment by luisherranz** [comment]
I've just released a new version with this and other fixes.

Thanks a lot @Bowriverstudio!! 🎉🎉😄

https://www.npmjs.com/package/frontity-chakra-theme
